inline.h

```diff
@@ -61,6 +61,19 @@ Perl_av_count(pTHX_ AV *av)
     return AvFILL(av) + 1;
 }
 
+PERL_STATIC_INLINE SSize_t
+Perl_AvFILL_(pTHX_ AV *av)
+{
+    PERL_ARGS_ASSERT_AVFILL_;
+    assert(SvTYPE(av) == SVt_PVAV);
```

Comment:
`PERL_ARGS_ASSERT_AVFILL_` already includes this assert. Sorry, didn't notice this the first time (forgot about 2463f19365f)

Reply:
Thanks; corrected

I note though that there are many other instances in core that have redundant asserts.  A quick grep yielded 70ish possible ones.